Rubberduck.Parsing/Grammar/VBA.g4

```diff
@@ -74,7 +74,9 @@
 *   - implemented proper specs for DATELITERAL.
 *   - added comments to parse tree (removes known limitation #2).
 *   - macroConstStmt now allowed in blockStmt.
-*   - allow type hints for parameters.
```

Comment:
Yeah, well... it was there (up to v1.2) when I grabbed the grammar file.. I just kept it going.